thatdevhuddy.md

```diff
@@ -0,0 +1,42 @@
+---
+name: "@thatdevhuddy"
```

Comment:
Could you add the names of your teammates here? 

Format: `name: ["@wahoo", "@hooray", "@etc"]`